h2o-r/h2o-package/R/kvstore.R

```diff
@@ -236,13 +239,26 @@ h2o.getModel <- function(model_id) {
       # Response column needs to be parsed
       if (name == "response_column")
         value <- value$column_name
-      allparams[[name]] <<- value
+        
+      if (effective)
+        effective_allparams[[name]] <<-value
```

Comment:
having this function modify its outside scope is not very good design, also <<- operator should not be used
I think using llply should do the trick